### PR TITLE
type-connect part 1: small stuff, generated stuff

### PR DESCRIPTION
**Pull request by chrisnojima**
This is a 9 part PR to improve the typing of our connected components. This has revealed some thing that were going untyped. I split it up into chunks to aid in reviewing, but only part 9 will pass flow/lint 

https://github.com/keybase/client/pull/13356
https://github.com/keybase/client/pull/13357
https://github.com/keybase/client/pull/13358
https://github.com/keybase/client/pull/13359
https://github.com/keybase/client/pull/13360
https://github.com/keybase/client/pull/13361
https://github.com/keybase/client/pull/13362
https://github.com/keybase/client/pull/13363

-----------

- [x] We kept 2 copies of the generated protocol js files. Now just keep the copy we keep in shared
- [x] export typePrefix from generated actions. Useful to get all routeTree actions for example
- [x] Remove entities/routeTree custom action creators and use json + generation instead (in prep for later parts of this PR where you can no longer dispatch non-generated actions)
- [x] Moved retention policy types to their own file to avoid cycles with chat/teams
- [x] Split generated rpc files into .flow and implementation files to avoid cycles with TypedState and engine / substore using RPCTypes
- [x] Remove entirely stubbed out flow-typed files. It creates types for every single file in the projects which adds too much time to flow. Going back to using lib/flowinterface file instead

### CODE REVIEW

**Comment by chrisnojima** [comment]
@keybase/react-hackers this is ready

File: shared/actions/entities.js [hunk 1]
```diff
@@ -1,34 +0,0 @@
-// @flow
```
Comment (cjb) [inline shared/actions/entities.js]: Maybe removing this file is a mistake?

File: shared/constants/types/rpc-gregor-gen.js.flow [hunk 2]
```diff
@@ -2,19 +2,23 @@
 /* eslint-disable */
 
 // This file is auto-generated by client/protocol/Makefile.
-// Not enabled: calls need to be turned on in enabled-calls.json
+import type {CallEffect, Effect} from 'redux-saga'
```
Comment (akalin-keybase) [inline shared/constants/types/rpc-gregor-gen.js.flow]: where are the changes to the code that generates these files?

Comment (akalin-keybase) [reply shared/constants/types/rpc-gregor-gen.js.flow]: Ah, part 2

**Comment by chrisnojima** [comment]
fix flow slow merges
- [ ] actions/teams
- [ ] profile/proofs

**Pull request merged**